aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -802,18 +798,13 @@ impl AptosVM {
         entry_fn: &EntryFunction,
         _txn_data: &TransactionMetadata,
     ) -> Result<(), VMStatus> {
-        // Note: Feature gating is needed here because the traversal of the dependencies could
-        //       result in shallow-loading of the modules and therefore subtle changes in
-        //       the error semantics.
-        if self.gas_feature_version >= 15 {
-            let module_id = traversal_context
-                .referenced_module_ids
-                .alloc(entry_fn.module().clone());
-            session.check_dependencies_and_charge_gas(gas_meter, traversal_context, [(
-                module_id.address(),
-                module_id.name(),
-            )])?;
-        }
+        let module_id = traversal_context
```

Comment:
These are very recent gas changes. We can't simply remove the feature gating in this case.

Reply:
Offline you said we can if replay passes😄 ok, will revert

Reply:
All gas changes reverted